docs/resources/Tutorials/how-to-annotate-a-genome.md

```diff
@@ -0,0 +1,136 @@
+
+### Caveats: plants!
+
+For plant genomes, especially larger ones with high repeat content, obtaining a high-quality  whole-genome alignment with other species (even those closely related to it) can be challenging. This can lead to poorer performance in annotation transfer tools (TOGA, Liftoff) than other approaches. Thus, for any path through the decision tree that ends with an annotation transfer method, one should confirm through a number of quality assessment metrics, in particular BUSCO scores (see below) that the annotation is of reasonably good quality.
+
+### Integrating methods: a work in progress
+
+If, after testing a few different methods, you discover that there is some degree of complementarity in the recovery of conserved single-copy orthologs, *i.e.* BUSCOs, [see this paper](https://academic.oup.com/bioinformatics/article/31/19/3210/211866) for more details. The current challenge is *how* to go about doing this. There are few currently maintained tools for combining an arbitrary number of gtf/gff3 files into a unified annotation. One example is [Mikado](https://github.com/EI-CoreBioinformatics/mikado) although our initial exploration of it a few years ago indicated that the scoring scheme for picking high quality transcripts may lead to the erroneous removal of real transcripts, and a subsequent loss of BUSCOs. At a future date we will explore the integration problem more deeply, and perhaps even come up with a solution! In the meantime, Mikado is worth looking at. A simpler, different approach that will not lead to any data loss would be to add annotations from one method that are non-overlapping with a "base" annotation (the entirety of which you will retain) to that base annotation:
+
+1. Given two annotation files for your genome, set one as the "base", i.e the one you want to which you want to add additional annotations
+2. Use [bedtools](https://bedtools.readthedocs.io/en/latest/) to identify unique annotations in your second annotation method, i.e. those that do not overlap the genomic coordinates of your "base" annotation. 
+3. Add those non-overlapping annotations to the "base". In principle, one does this at the gene-level, adding non-overlapping genes and their respective child features to the "base" annotation. 
+4. If you have more than two annotations to integrate, set the initially integrated annotation as your base, and follow the steps above with the next annotation.
+
+Whichever method you use for integrating annotations from multiple methods, a closer look at the decision tree to understand recommended combinations of tools to integrate is warranted. These decisions reflect a desire to avoid redundancy between tools that either use the same type of evidence, or make the same sort of predictions, yet with one tools clearly having poorer performance at achieving the same task. Options are as follows:
```

Comment:
I added "is warranted" at the end of this sentence because it seemed incomplete, but make sure I preserved the intended meaning.